### DISCUSSION TITLE
HA und RaspberryMatic in Proxmox Umgebung

### DISCUSSION THREAD

**Discussion by AFPass** [OpenCCU/OpenCCU#1986] [🇩🇪 Fragen&Antworten]
Hallo,

ich habe HA als VM in einer Proxmox Umgebung installiert.
HA läuft in einer Debian VM
Nun möchte ich gerne noch RaspberryMatic in HA einbinden.

Nach der Anleitung https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant bin ich vorgegangen.

Ich scheitere an folgender Stelle:
apt install raspberrypi-kernel-headers 

Ich bekomme folgende Meldung:
`Paketlisten werden gelesen… Fertig
Abhängigkeitsbaum wird aufgebaut… Fertig
Statusinformationen werden eingelesen… Fertig
Paket raspberrypi-kernel-headers ist nicht verfügbar, wird aber von einem anderen Paket
referenziert. Das kann heißen, dass das Paket fehlt, dass es abgelöst
wurde oder nur aus einer anderen Quelle verfügbar ist.

E: Für Paket »raspberrypi-kernel-headers« existiert kein Installationskandidat.`

Wie bekomme ich die Kernel Headers installiert?

**Comment by Baxxy13** [comment]
Warum machst du dir das so unnötig kompliziert?

> HA läuft in einer Debian VM

Besser wäre es Homeassistant-OS als eigenständige VM aufzusetzen, dann bräuchtest du nur deine per USB angebundene Funkmodullösung an die VM durchreichen. Die "Supervised-Installation" ist für Komplikationen bekannt. 

Noch besser wäre es RaspberryMatic als eigenständige VM aufzusetzen. Die USB-Funkmodullösung dann an diese "RM-VM" durchreichen und gut ist's. Dazu dann eine separate Homeassistant VM (egal ob OS oder Supervised).

> Wie bekomme ich die Kernel Headers installiert?

Gegenfrage: 
Was soll eine VM mit Raspberry Kernel Headern? Proxmox ist doch ein X86-64 System.

**Comment by AFPass** [comment]
Danke für deine Antwort. 

OS hatte ich auch probiert. 
Da komme ich aber nicht mit der Installation von vzlogger klar. 
Daher habe ich die Supervised Version genommen. 

Mir hat der Gedanke von HA und HM unter einem Dach gefallen.

**Reply by Baxxy13** [reply]
> Mir hat der Gedanke von HA und HM unter einem Dach gefallen.

Ich rate von ab. Wie geschrieben ist die Supervised-Installation für Probleme in Zusammenhang mit dem RM-HA-AddOn bekannt.
Auch das OS kann problematisch sein wie kürzlich der Switch auf 9.0 zeigte. Für jedes System eine eigene VM ist m.E. die beste Option.

**Reply by AFPass** [reply]
Hm, ok. Danke.
